procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -257,7 +257,7 @@
    "source": [
```

Comment:
<p>Does that link work for you?</p><p>It was broken for me when I tried it the other day.</p><p>If it is, please just fix it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Great catch! Thanks!</p><p> </p>